jaws-journey-deploy/orb.yml

```diff
@@ -49,6 +50,11 @@ aliases:
     var_file: << parameters.environment >>.tfvars
     label: << parameters.environment >>
 
+  slack-notification: &slack-notification
+    event: fail
+    mentions: '@jaws-support'
```

Comment:
Wondering if the mentions should be a passed variable - just in case we have multiple support teams in the future? - Probably not though and it would make updating hard if we changed the link.

Reply:
might be something to look at in the future